swift/ql/lib/codeql/swift/elements/expr/DeclRefExpr.qll

```diff
@@ -1,5 +1,9 @@
 private import codeql.swift.generated.expr.DeclRefExpr
 
 class DeclRefExpr extends Generated::DeclRefExpr {
-  override string toString() { result = this.getDecl().toString() }
+  override string toString() {
+    if exists(this.getDecl().toString())
```

Comment:
Future work: We should check if we can actually do with just `if exists(this.getDecl())`. I'm pretty sure every declaration has a `toString`, so that should be equivalent, right?

Reply:
Yeah, that's pretty much always the case, except when we are referencing a missing Builtin.